api/parsers.go

```diff
@@ -99,10 +99,10 @@ func (p *Parsers) Add(repositoryName string, parser *Parser, force bool) error {
 }
 
 func (p *Parsers) Get(repositoryName string, parserName string) (*Parser, error) {
-
 	var query struct {
 		Repository struct {
 			Parser *struct {
+				Id         string
```

Comment:
`Id` should be `ID` by convention in Go.

```suggestion
				ID         string
```